tasks/bootstrap-deploy-user.yml

```diff
@@ -25,20 +25,15 @@
   authorized_key:
     user: "{{ sb_debian_base_deploy_user }}"
     key:  "{{ item }}"
-  with_file:
-    - keys/deploy_users
-    - keys/ci-{{ sb_debian_base_app_environment }}
+  with_items: "{{ sb_debian_base_deploy_user_authorized_keys }}"
 
-- name: Add SSH keys for GitHub's "Deploy keys"
-  template:
-    src:  "{{ item.src }}"
-    dest: "{{ item.dest }}"
-    mode: "{{ item.mode }}"
+- name: Add deploy user private key
+  copy:
+    dest:  "{{ sb_debian_base_deploy_user_dir }}/.ssh/id_rsa"
+    content: "{{ sb_debian_base_deploy_user_private_key }}"
```

Comment:
I agree, we shouldn't define any SSH keys as default. When those variables are not defined, Ansible will interrupt and exit with an error, we don't need to do any further checks here.